### PR TITLE
manual: rename section

### PR DESCRIPTION
**Pull request by tjhei**
after longer discussion

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -330,7 +330,7 @@ \subsection{Acknowledgments}
   \end{center}
 }
 
-\section{Equations, models, coefficients}
+\section{Geodynamic Models and Numerical Methods in \aspect{}}
```
Comment (bangerth) [inline doc/manual/manual.tex]: For consistency with all other headings, don't capitalize any but the first word.

**Comment by tjhei** [comment]
> Out of curiosity, how did you come up with "Geodynamic models"? That seems a rather broad term.

It is the best name we came up with that describes how we model the problem as a PDE and what approximation we use (density parameterization etc).

**Comment by bangerth** [comment]
It seems too generic to me. Why not "Models for mantle convection"?

File: doc/manual/manual.tex (hunk 1 above)
Comment (bangerth) [reply doc/manual/manual.tex]: @tjhei -- ping?

**Comment by gassmoeller** [comment]
Because we are not longer only a code for mantle convection. What about "Geodynamic modelling assumptions and numerical methods"? We discussed a larger number of different options, and this was the best we came up with.

**Comment by bangerth** [comment]
Yes, I like that title better. Can you make the change and merge yourself?

**Comment by gassmoeller** [comment]
I would like @tjhei or @jdannberg to look over this and make the decision on merging it.

**Comment by gassmoeller** [comment]
Could someone just push the merge button? I am happy with the current state, but I modified it, and dont want to merge my own change.

**Pull request merged**